workers/httpsrv/httpsrv.go

```diff
@@ -82,6 +85,18 @@ func HealthHandler(now func() time.Time) http.HandlerFunc {
 
 }
 
+// CORSHandler handles pre-flight CORS requests.
+var CORSHandler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodOptions {
+		log.Printf("CORS pre-flight aborted: %s!=OPTIONS", r.Method)
+		return
+	}
+	w.Header().Set("Access-Control-Allow-Origin", "*")
+	w.Header().Set("Access-Control-Allow-Headers", "Authorization, Content-Type, X-Service-Version, Cache-Control")
```

Comment:
`x-service-version` should not be part of this list.

Headers beginning with an `x` denote a custom header.

Reply:
Oops. Lifted these from the api gateway and didn’t notice that one. Lazy